source/includes/_08-additional-features.md.erb

```diff
@@ -479,4 +480,115 @@ should be surfaced using the <code>field_checks</code> attribute of the <code>ch
       <td>The result of the comparison. One of <code>["MATCH", "MISMATCH"]</code></td>
     <tr>
   </tbody>
-</table>
\ No newline at end of file
+</table>
+
+## Embedding or Linking an External Webpage
+
+Integrations have the option to include a URL within the Check Result. This URL can then either be displayed as a link, or used to embed the resulting webpage within Passfort.
+
+The identifier for this feature is either <code>EXTERNAL_LINK</code> or <code>EXTERNAL_EMBED</code> depending on how you intend to surface the external resource. This feature is compatible with all of our supported check types.
+
+The `external_resources` object includes whether the external resource should be displayed as a link or an embed, the url, id, and a label describing the resource. 
```

Comment:
"should" to "is"

Reply:
Hmm...I suggest adding a colon after "includes" to make it clear we're introducing a list